docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation with shelfLocator (non-UTK)
+--------------------------------------------
+
+Use Case
+^^^^^^^^
+Instances where non-UTK held items have shelfLocator information.
+
+Justification
+^^^^^^^^^^^^^
+While, we do not not know if this shelfLocator information is accurate, we will opt to retain it going forward as a string and map to skos:note. Samvera does note some possible future availability of opaque:locationShelfLocator, however this predicate does not exist yet.
+
+Xpath
+^^^^^
+
+`mods:location/mods:shelfLocator` OR other variations...
+
+Decision
+^^^^^^^^
+We will retain `mods:shelfLocator` data when present for non-UTK records, and transcribe this to a skos:note.
+
+`Example record from volvoices:2136 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2136/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Cleveland State Community College</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Photograph Collection 2, People</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Cleveland</text>
+                    <text>County: Bradley County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "Cleveland State Community College" ;
+        skos:note "Shelf locator: Photograph Collection 2, People" .
+
+physicalLocation with holdingExternal
+-------------------------------------
+
+Use Case
+^^^^^^^^
+Some instances in our collections contain nested subelements for holdingExternal and the further nested physicalAddress information.
+
+Justification
+^^^^^^^^^^^^^
+To keep our metadata as simple as possible from a technical standpoint we will drop all information for holdingExternal. This type of information has little additive value when physicalLocation is already referenced.
+
+Xpath
+^^^^^
+
+`mods:location/mods:holdingExternal`
```

Comment:
We made the decision a while ago to treat all xpath sections as inline code.  Switch to:

:code:`mods:location/mods:holdingExternal`